src/dispatch-payload.ts

```diff
@@ -111,6 +116,13 @@ export const DispatchPayload = async (
   return resp || {};
 };
 
+// deno-lint-ignore no-explicit-any
+function isAllowNetError(e: any): boolean {
+  return e?.name === "PermissionDenied" &&
+    typeof e?.message === "string" &&
+    e?.message.includes("run again with the --allow-net flag");
```

Comment:
Do you think there is room to pare this down a bit? I'm imagining a Deno update that changes the verbiage on this error will give us grief here. Could get away with a smaller substring like just `e?.message.includes("--allow-net")`?

Reply:
now only checks --allow-net